profiles/fscloud/README.md

```diff
@@ -0,0 +1,6 @@
+# Financial Services Cloud Profile
+
+This is a profile for PostgreSQL that meets Financial Services Cloud requirements.
+It has been scanned by [IBM Code Risk Analyzer (CRA)](https://cloud.ibm.com/docs/code-risk-analyzer-cli-plugin?topic=code-risk-analyzer-cli-plugin-cra-cli-plugin#terraform-command) and meets all applicable goals with the following exception:
+
+> Rule ID rule-3b2768e5-d783-4b0c-a47f-81479af34689 : Check whether Event Streams is accessible only by using private endpoints
```

Comment:
This is missing the metadata tags for terradocs. Please add it, and have it consistent with https://github.com/terraform-ibm-modules/terraform-ibm-icd-postgresql/blob/main/profiles/fscloud/README.md

Reply:
Done

Reply:
I would add a line under this to explain that this rule fails due to a known CRA bug